en/docs/insights/derive-insights.md

```diff
@@ -0,0 +1,364 @@
+    - The maximum number of alerts that can be configured is 20 for each organization, environment, and tenant combination.
+    - Adding an email to an alert configuration is optional. The maximum number of emails per alert configuration is limited to 5.
+    
+The alert can be a latency alert or a traffic alert.
+
+#### Latency alerts
+
+Configure latency alerts to be notified if the response latency of APIs is greater than a predefined threshold. This is useful when you have APIs that should honor SLAs and when you want to know about slow APIs proactively.
+
+To configure a new latency alert, follow the steps below:
+
+1. In the Insights Portal, click **Alert Configuration** to open the **Alert Configuration** page.
+
+    The **Latency** tab is open by default.
+
+2. Verify that you are in the correct organization, and select the required environment. 
+
+3. In the **API Name** field, select the API for which you want to configure the alert.
+
+    !!! Info
+        Only the APIs that you have invoked at least once are listed here. For other APIs, you need to give the API name in the required format as instructed in the UI.
+        
+4. In the **Metric** field, select the required metric against which you want to evaluate the alert configuration.
+
+    !!! tip
+        The list includes all available options. If there are multiple metrics, you can select the required metric. If there is only one metric to select, that metric is selected by default, and the field is disabled. 
+
+5. In the **Latency** field, specify the threshold in milliseconds.
+
+    !!! info
+        When the 95th percentile of the selected metric exceeds the threshold provided here, alerts are triggered.
+        
+6. In the **Emails** field, specify the list of emails that should be notified when the alert is generated.
+        
+7. Click **Add**.
+
+Once an alert is successfully added, the alert configuration is shown in the table as follows. Each configuration can be edited and removed using this table. If multiple emails are configured, they are displayed in a comma-separated list.
+
+#### Traffic alerts
+
+Configure Traffic alerts to identify if the request count of APIs is greater than a predefined threshold. This is useful when you have APIs that are using backends with traffic limits or monetized backends that require your system to proactively scale depending on the incoming traffic.
+
+To configure a new latency alert, follow the steps below:
+
+1. In the Insights Portal, click **Alert Configuration** to open the **Alert Configuration** page.
+
+2. Click **Traffic** to open the **Traffic** tab.
+
+3. Verify that you are in the correct organization, and select the required environment. 
+
+4. In the **API Name** field, select the API for which you want to configure the alert.
+
+    !!! Info
+        Only the APIs that you have invoked at least once are listed here. For other APIs, you need to give the API name in the required format as instructed in the UI.
```

Comment:
shall we mention the format here?

Reply:
The format is given below the field in the UI itself.

Reply:
well if we are saying "required format ", we could make it much more clearer and tell the user exactly what the format is. That will help the reader, don't you think?

Reply:
It says "the required format as specified in the UI".

Then the UI gives the exact format as shown below.

<img width="332" alt="Screenshot 2021-09-09 at 12 55 22" src="https://user-images.githubusercontent.com/26239861/132641881-bbb5e196-a2b5-4241-962e-9082f82d08ab.png">

This way, even if the supported format changes, the instruction in the documentation won't be outdated.